src/Component/EventSourcing/Domain/AggregateHistory.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+
+namespace Ubirak\Component\EventSourcing\Domain;
+
+final class AggregateHistory extends \SplFixedArray
+{
+    private $aggregateId;
+
+    private $versionId;
+
+    public function __construct(iterable $events, ?int $versionId = null)
```

Comment:
Why not pute int type for versionId and a 0 default value?

Reply:
yep I will do it